src/søknad/steg/1-omdeg/sivilstatus/Sivilstatus.tsx

```diff
@@ -63,14 +62,9 @@ const Sivilstatus: React.FC<Props> = ({
         verdi: svar,
       },
     };
-    if (
-      spørsmål.søknadid === ESivilstatusSøknadid.harSøktSeparasjon &&
-      harSøktSeparasjon?.verdi === false &&
```

Comment:
Hvorfor kan `harSøktSeparasjon` fjernes her?

Reply:
Vi vil nullstille `datoSøktSeperasjon` og `datoFlyttetFraHverandre` uansett når man velger "Nei", så de to datoene ikke blir sendt med videre. Koden som var der fra før nulstiller først når man trykker på "Ja" igjen, og dermed blir datoene med potensielle ugyldige verdier sendt med, selvom de ikke har kysset av for søkt seperasjon.
![image](https://user-images.githubusercontent.com/21220467/129680995-89bb4ce9-8b90-4a92-8f9f-1f17c338661a.png)